database/sql/V021__store_full_quotes.sql

```diff
@@ -0,0 +1,27 @@
+-- This migration step will change how we store min-fee measurements to store
+-- full quote data that can later be attached to an order.
+--
+-- This involves adding new columns for buy and sell amounts in the quotes as
+-- well as renaming the tables so its more apparent what they are for.
+-- Additionally, for the quotes table, we add a `bigserial` ID column for
+-- generating unique quote IDs for each order.
+
+ALTER TABLE min_fee_measurements RENAME TO quotes;
```

Comment:
I assume we have some queries outside of this repo which rely on those table names.
Maybe check with Ben or so if/how much that would break.

Reply:
I don't think this is used for anything significant and we don't provide "a stable api" to anything outside this repository at the moment. Still good to keep this in mind but I wouldn't let it block any change.

Reply:
`min_fee_measurement`s were already disposable (in that they would be removed when expired).

The `order_fee_parameters` tables is accessible through the `GET api/v1/orders/{orderUid}` API. The query for populating the fee data there was adjusted to account for the new table name.

Reply:
The [settlement analysis tool](https://github.com/cowprotocol/settlement-analysis) uses that table: https://github.com/cowprotocol/settlement-analysis/blob/main/src/main.rs#L203.
Of course no major issue but we should be aware of such things to not have to remember + debug such things when actually need those tools.

Reply:
I'll make a PR there.